ci/docker/Dockerfile.build.jetson

```diff
@@ -77,6 +77,7 @@ RUN JETPACK_DOWNLOAD_PREFIX=https://developer.download.nvidia.com/devzone/devcen
     dpkg -i --force-architecture  $ARM_NVINFER_INSTALLER_PACKAGE && \
     dpkg -i --force-architecture  $ARM_NVINFER_DEV_INSTALLER_PACKAGE && \
     apt update -y || true && apt install -y cuda-libraries-dev-9-0 libcudnn7-dev libnvinfer-dev
+RUN cd /usr/include/aarch64-linux-gnu/ && ln -s cudnn_v7.h cudnn.h
```

Comment:
why not?

Reply:
Because that adds additional side effects to this command. If there's no reason to introduce side effects, you generally would want to avoid it. For that symlink, I don't see much reason.
Otherwise, follow up commands might now write to that new path since they assume an unchanged path.

Reply:
I asked "why not?" meaning, sure why not. I agreed with you, sorry for the misunderstanding.